distributed/diagnostics/progress.py

```diff
@@ -291,3 +294,80 @@ def transition(self, key, start, finish, *args, **kwargs):
     def restart(self, scheduler):
         self.all.clear()
         self.state.clear()
+
+
+class GroupTiming(SchedulerPlugin):
+    """Keep track of high-level timing information for task group progress"""
+
+    name = "group-timing"
+
+    def __init__(self, scheduler):
+        scheduler.add_plugin(self)
+        self.scheduler = scheduler
+
+        # Time bin size. TODO: make this configurable?
+        self.dt = 1.0
```

Comment:
thats `ms` or how can one interpret this?

Reply:
This is in seconds, I'll note it in the code